src/include/migraphx/module.hpp

```diff
@@ -182,6 +184,7 @@ struct module
 
     instruction_ref validate() const;
     instruction_ref find_dangling_reference() const;
+    instruction_ref flag_division_by_zero() const;
```

Comment:
instead of `flag_division` let's use `find_division_by_zero` for consistency.

Reply:
done